bin/drop_sample_annot.py

```diff
@@ -50,24 +55,78 @@ def is_paired_end(single_end: str) -> bool:
     return False
 
 
+def is_stranded(strandedness: str) -> str:
+    """Logical funciton to determine sample strandness"""
+    if strandedness.lower() == "reverse":
+        return "reverse"
+    elif strandedness.lower() == "forward":
+        return "yes"
+    else:
+        return "no"
+
+
+def count_mode(sample_count_mode: str) -> str:
+    """Logical funciton to determine if count mode is given or default "IntersectionStrict" should be used"""
+    print("Hello")
```

Comment:
Should it say something else than Hello?

Reply:
Forgot to remove the prints, they were there just to make sure it was run